site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
 ---
 layout: "layouts/doc-post.njk"
-title: Set up Group Publishing
+title: Set up group publishing
 date: 2020-06-20
 description: How to share ownership of your Chrome Web Store items with other developers.
 ---
 
-You can share ownership of your items in Google Chrome Web Store with other developers by setting up
-group publishing. With group publishing, you can add developers to a Google Group, who can then act
-on your behalf. They'll have access to all the items you own and can make any changes to them that
-you can make.
+You can share ownership of your items in Google Chrome Web Store with other developers by setting up group publishing.
+
+This guide tells you how to set up group publishing.
+
+## About group publishers
+
+Use a group publisher to establish an entity that owns Chrome Web Store items, so that publishing isn't tied to any individual developer. Consider how this differs from the more basic individual publisher role.
+
+**Individual publisher** When an individual developer acts as a publisher of an item, only that developer can upload and publish updates to the item. The following diagram describes this scenario:
```

Comment:
do we need to mention "when you register as a Chrome Web Store developer, an individual publisher account is automatically created and associated with your developer account"?

Reply:
Done